packages/manager/modules/pci/src/projects/project/failover-ips/onboarding/order/order.routing.js

```diff
@@ -19,22 +15,50 @@ export default /* @ngInject */ ($stateProvider, coreConfigProvider) => {
     resolve: {
       breadcrumb: /* @ngInject */ () => null,
       /* @ngInject */
-      products: (OvhApiOrderCatalogFormatted) =>
+      products: /* @ngInject */ (
+        availableRegions,
+        OvhApiOrderCatalogFormatted,
+      ) =>
         OvhApiOrderCatalogFormatted.v6()
           .get({
             catalogName: 'ip',
             ovhSubsidiary: coreConfigProvider.getUser().ovhSubsidiary,
           })
           .$promise.then(({ plans }) =>
-            filter(
-              plans,
+            plans.filter(
               (offer) =>
                 /failover/.test(offer.planCode) &&
-                offer.invoiceName.includes(
-                  get(REGIONS, coreConfigProvider.getRegion()),
+                availableRegions.some((region) =>
+                  offer.invoiceName.includes(region),
                 ),
             ),
           ),
+      continentCodeEnum: /* @ngInject */ ($http) =>
+        $http
+          .get('/cloud.json')
+          .then(
+            ({ data: schema }) =>
+              schema.models['cloud.RegionContinentEnum'].enum,
+          ),
+      availableRegions: /* @ngInject */ (
```

Comment:
This could have been set in an parent routing but as I request the enum (which slow down loading), I choose to duplicate it to sections that requires it